BlueShift-iOS-Extension-SDK/BlueShiftPushAnalytics.m

```diff
@@ -0,0 +1,122 @@
+//
+//  BlueShiftPushAnalytics.m
+//  BlueShift-iOS-Extension-SDK
+//
+//  Created by shahas kp on 11/10/17.
+//
+
+#import "BlueShiftPushAnalytics.h"
+#import "SDKVersion.h"
+#import "BlueShiftPushNotification.h"
+
+#define kBaseURL                        @"https://api.getblueshift.com/"
+#define kPushEventsUploadURL            @"track"
+#define kStatusCodeSuccessfullResponse  200
+
+@implementation BlueShiftPushAnalytics
+
++ (void)sendPushAnalytics:(NSString *)type withParams:(NSDictionary *)userInfo {
+    NSDictionary *pushTrackParameterDictionary = [self pushTrackParameterDictionaryForPushDetailsDictionary:userInfo];
```

Comment:
create the parameters dictionary required for event api from the push notification payload which received.